From d94422ad1047ccfb00a960c8ad9ce8697e63334f Mon Sep 17 00:00:00 2001
From: Ky Nguyen <ky.nguyen.uj@renesas.com>
Date: Fri, 23 Aug 2019 14:10:27 +0700
Subject: [PATCH 226/328] drivers: clk: renesas: r8a774b1: Define clock module
 for TPU

This patch defines clock module for TPU for R8A774B1 SoC.

Signed-off-by: Ky Nguyen <ky.nguyen.uj@renesas.com>
---
 drivers/clk/renesas/r8a774b1-cpg-mssr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/renesas/r8a774b1-cpg-mssr.c b/drivers/clk/renesas/r8a774b1-cpg-mssr.c
index eb3f326..cfb81cb 100644
--- a/drivers/clk/renesas/r8a774b1-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a774b1-cpg-mssr.c
@@ -135,6 +135,7 @@ enum clk_ids {
 	DEF_MOD("cmt2",			 301,	R8A774B1_CLK_R),
 	DEF_MOD("cmt1",			 302,	R8A774B1_CLK_R),
 	DEF_MOD("cmt0",			 303,	R8A774B1_CLK_R),
+	DEF_MOD("tpu",			 304,	R8A774B1_CLK_S3D4),
 	DEF_MOD("scif2",		 310,	R8A774B1_CLK_S3D4),
 	DEF_MOD("sdif3",		 311,	R8A774B1_CLK_SD3),
 	DEF_MOD("sdif2",		 312,	R8A774B1_CLK_SD2),
-- 
1.9.1

